recipe/meta.yaml

```diff
@@ -24,7 +24,7 @@ requirements:
   run:
     - python >=3
     - nbconvert >=5.5.0,<6
-    - jupyter_server ==0.1.0,<0.2.0
+    - jupyter_server >=0.1.0,<=0.1.1,<0.2.0
```

Comment:
Is there an issue with making it:

```
- jupyter_server >=0.1.0,<0.2.0
```
That way you get 0.1.0, 0.1.1, 0.1.2, etc., up to but not including 0.2.0

Reply:
Of course. My concern was that someone had previously chosen `==0.1.0` (which is strange, as it does make `<0.2.0` redundant. And so could have been a typo), and I didn't want to second guess that choice.

But either way, everyone keeps telling me it should be `jupyter_server >=0.1.0,<0.2.0`, so this PR is really to fix that now 😄